js/w3c/style.js

```diff
@@ -73,15 +73,16 @@ define(
             case "LC-NOTE":
               styleFile += "WD";
               break;
+            case "WG-NOTE":
             case "FPWD-NOTE":
-              styleFile += "WG-NOTE";
+              styleFile += "WG-NOTE.css";
```

Comment:
Actually it works fine without the .css.  I just tried it.  What doesn't work is http: - it redirects to https.